File: official/docs/ruby/scan_forms/create_scanform.rb
require 'easypost'

EasyPost.api_key = ENV['EASYPOST_API_KEY']

scan_form = EasyPost::ScanForm.create(
  shipments: [
    {
      id: 'shp_...',
    },
    {
      id: 'shp_...',
    },
  ],
)

puts scan_form

File: official/docs/csharp/scan_forms/create-scanform.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

            EasyPost.ClientManager.SetCurrent(apiKey);

            Shipment shipment = await Shipment.Retrieve("shp_...");

            ScanForm scanForm = await ScanForm.Create(new List<Shipment>
            {
                shipment
            });

            Console.WriteLine(JsonConvert.SerializeObject(scanForm, Formatting.Indented));
        }
    }
}